extensions/amp-tiktok/amp-tiktok.md

```diff
@@ -0,0 +1,157 @@
+applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# amp-tiktok
+
+<!--
+  If the component is relevant for more than one format and operates differently between these
+  formats, include and filter multiple content blocks and code samples.
+-->
+
+## Usage
+
+The simplest usage of the `amp-tiktok` component has the `width`, `height` and `data-src` attributes. This will display the Tiktok embed inside of an iframe on the page.
+
+```html
+<amp-tiktok
+  width="325"
+  height="575"
+  data-src="6718335390845095173"
+></amp-tiktok>
+```
+
+`amp-tiktok` does not support autoplay and requires user interaction to play videos.
+
+### Behavior
+
+## Avoiding Layout shift
+
+The `width` and `height` attributes determine the initial laid out height of the Tiktok embedded in the page.
+
+However, we set the width of the Tiktok iframe player to be 325px wide. This results in a height of roughly 575px for the video player, however the full height of the player depends on the length and content of the Tiktok caption. (These values represent current behavior and subject to change).
+
+In order to show the entire height of the video, this component will resize its height to include the full player height. Therefore in order to avoid layout shift it is best to match the `width` and `height` attributes to those values.
+
+[example preview="inline" playground="true" imports="amp-tiktok"]
+
+```html
+<amp-tiktok
+  width="325"
+  height="731"
+  data-src="6718335390845095173"
+></amp-tiktok>
+```
+
+[/example]
+
+If you choose to set the width to a value which is greater than 325px, the `iframe` will remain 325px and will be horizontally centered in that space. The surrounding space will be empty.
+
+If you choose to set the height of to a value which is which is greater than the height the height will remain the height of the `iframe`.
+
+## Attributes
+
+### `data-src`
+
+The `data-src` attribute can contain one of two values: a **video id** or a **full URL** to a Tiktok detail page.
```

Comment:
```suggestion
The `data-src` attribute can contain one of two values: a **video id** or a **full URL** to a TikTok detail page.
```

Reply:
In the TikTok branding they capitalize both T's should we be doing the same? or is 'Tiktok' more correct?

Reply:
I usually follow the company branding. I imagine seeing `Tictok` is a lot like seeing `Youtube` - everyone will understand, but its slightllllly off. lol

Reply:
Sounds good! Thanks!